src/SchematicHQ.Client/Webhooks/WebhookUtils/WebhookVerifier.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace SchematicHQ.Client.Webhooks.WebhookUtils
+{
+    /// <summary>
+    /// Utilities for verifying Schematic webhook signatures.
+    /// </summary>
+    public static class WebhookVerifier
+    {
+        /// <summary>
+        /// Header containing the webhook signature.
+        /// </summary>
+        public const string WebhookSignatureHeader = "X-Schematic-Webhook-Signature";
```

Comment:
I can't remember if CSharp allows you do put constants outside a class, but these might be better suited at namespace scope.

Reply:
Thinking more about this - and this goes for every official SDK - if we want to gather all the valid headers globally, we can probably import and re-export them in a dedicated namespace. Not a blocker here.